test/blackbox-tests/test-cases/melange/emit-installed.t

```diff
@@ -0,0 +1,62 @@
+Test dependency on installed package
+
+  $ mkdir a b prefix
+
+  $ cat > a/dune-project <<EOF
+  > (lang dune 3.6)
+  > (package (name a))
+  > (using melange 0.1)
+  > EOF
+  $ cat > a/dune <<EOF
+  > (library
+  >  (modes melange)
```

Comment:
Is there a test that has `(modes melange byte)`? To make sure they don't step over each other.